src/resolvers/donationResolver.ts

```diff
@@ -279,6 +279,7 @@ export class DonationResolver {
         transactionNetworkId: Number(transactionNetworkId),
         currency: token,
         projectWalletAddress: project.walletAddress,
+        segmentNotified: true,
```

Comment:
@CarlosQ96 
When creating donations we should set this to true, otherwise if user create donation, we will send him an email but because we didn't set this field to `true`, the cronjob will send another one

Reply:
Agreed, thanks !